addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/discovery/FreeboxServerDiscoveryParticipant.java

```diff
@@ -71,12 +72,15 @@ public DiscoveryResult createResult(ServiceInfo service) {
         if (thingUID != null && ip != null) {
             logger.info("Created a DiscoveryResult for Freebox Server {} on IP {}", thingUID, ip);
             Map<String, Object> properties = new HashMap<>(1);
-            properties.put(FreeboxServerConfiguration.IP_ADDRESS, ip);
+            properties.put(FreeboxServerConfiguration.IP_ADDRESS, ip + ":" + service.getPort());
```

Comment:
In fact this parameter contains a FQDN, it can be 192.168.0.254:80 but it can be mafreebox.free.fr.
Maybe the name of the parameter could have been different.